mdb-realm/src/deploy.ts

```diff
@@ -0,0 +1,28 @@
+import * as core from "@actions/core";
+import { createCluster, getConfig } from "./helpers";
+
+async function run(): Promise<void> {
+    try {
+        const config = getConfig();
+        for (let i = 0; i < 5; i++) {
+            try {
+                await createCluster(config);
```

Comment:
Should there be a small wait time before each try

Reply:
Those are http calls, so they inherently incur some delays. The intention here is to handle a case where two steps attempt to deploy a cluster at the same time - one of them will fail and when it retries it will see the cluster the other one deployed and use that instead.